lammps-tutorials.tex

```diff
@@ -268,81 +262,87 @@ \subsection{Scope}
+atomic interactions \cite{van2001reaxff}.  ReaxFF includes charge equilibration
+(QEq), a method that allows the partial charges of atoms
+to adjust according to their local environment.
 
 In \hyperref[gcmc-silica-label]{tutorial 6}, a Monte Carlo simulation in
-the Grand Canonical ensemble is implemented to illustrate the use of
-LAMMPS for simulating an open system that can exchange particles with a
-reservoir.
+the Grand Canonical ensemble is implemented to demonstrate how
+LAMMPS can be used to simulate an open system that exchange
+particles with a reservoir.
 
 In \hyperref[umbrella-sampling-label]{tutorial 7}, an advanced free
-energy method called umbrella sampling is implemented.  Through the
-calculation of an energy barrier, this tutorial describes a protocol
-that can be used to deal with energy landscapes that prevent their
-sampling through classical MD or MC methods.
+energy method called umbrella sampling is implemented.  By calculating
+an energy barrier, this tutorial describes a protocol
+for addressing energy landscapes that are difficult to sample using
+classical MD or MC methods.
 
-% AK: As an additional justification for type labels, there could be an
-% 8th tutorial added showcasing fix bond/react, possibly based on one of
-% examples bundled with LAMMPS. Jake?
+In \hyperref[bond-react-label]{tutorial 8}, a CNT embedded in
+nylon-6,6 polymer melt is simulated.  The
+REACTER protocol is used to model the polymerization of nylon, and the formation
+of water molecules is tracked over time~\cite{gissinger2020reacter}.
 
 \section{Prerequisites}
 
 \subsection{Background knowledge}
 
 This set of tutorials assumes no prior knowledge of the LAMMPS software
-itself.  Completing the tutorials requires a text editor and
-a suitable LAMMPS executable.  We use LAMMPS--GUI \cite{lammps_gui_docs}
-here since it has features that make it particularly convenient for
-tutorials, but other console or graphical text editors like GNU nano,
-vi/vim, Emacs, Notepad, Gedit, Visual Studio Code can also be
-used.  LAMMPS can be executed directly from LAMMPS--GUI
-or from the command line, which requires some familiarity with executing
-commands from a terminal or command--line prompt.
+itself.  To complete the tutorials, a text editor and a suitable LAMMPS
+executable are required.  We use LAMMPS--GUI~\cite{lammps_gui_docs}
+here, as it offers features that make it particularly convenient for
+tutorials, but other console or graphical text editors, such as GNU nano,
+vi/vim, Emacs, Notepad, Gedit, and Visual Studio Code, can also be
+used.  LAMMPS can be executed either directly from
+LAMMPS--GUI (\hyperref[using-lammps-gui-label]{Appendix~\ref{using-lammps-gui-label}})
+or from the command line (\hyperref[command-line-label]{Appendix~\ref{command-line-label}}),
+the latter of which requires some familiarity with executing commands
+from a terminal or command-line prompt.
 
 In addition, prior knowledge of the theoretical basics of molecular
```

Comment:
@akohlmey @jrgissing If you have other suggestion of book you like, please feel free to add it. I know there are other classic book out there, but I did not want to write the name of a book I did not open.

Reply:
I am refering to the paragraph starting with "In addition, prior knowledge of the theoretical basics of molecular ... "

Reply:
The main text books that I used while learning MD were the Allen/Tildesley and Hansen/McDonald books. In addition I used D. A. McQuarrie, Statistical Mechanics, Harper & Row, New York, 1976, but I think any decent text book on statistical mechanics or statistical thermodynamics will do. A good online (meta) resource would be http://www.sklogwiki.org/